guide/installation-procedure.md

```diff
@@ -137,6 +137,14 @@ or
 gdebi webots_{{ webots.version.major }}.{{ webots.version.minor }}.{{ webots.version.bugfix }}_amd64.deb
 ```
 
+#### Server edition
+
+Webots requires some graphical features that are usually not available by default on a linux server edition and additional packages need to be available to make it work.
+
+Webots depends on `xserver-xorg-core` and `libxslt1.1`. These packages are automatically installed when using the DEB package, but in case of the tarball package the user has to manually install them.
```

Comment:
Because this is a special case and only very few user will user a server edition where these two packages are not available.
In desktop edition this packages are installed by default.
It seems a pity to make the tarball package bigger just for very few users.
But I can include them in the tarball package.